applications/StructuralMechanicsApplication/custom_constitutive/linear_plane_strain.cpp

```diff
@@ -55,86 +55,133 @@ LinearPlaneStrain::~LinearPlaneStrain()
 {
 }
 
-void  LinearPlaneStrain::CalculateMaterialResponsePK2 (Parameters& rValues)
+void  LinearPlaneStrain::CalculateMaterialResponsePK2 (Parameters& rParameterValues)
```

Comment:
![imagen](https://user-images.githubusercontent.com/19991680/28660140-2ce9b012-72b2-11e7-9d08-c287e69a1385.png)

This is from constitutive_law.h